alternator/executor.hh

```diff
@@ -163,7 +166,7 @@ class executor : public peering_sharded_service<executor> {
     // An smp_service_group to be used for limiting the concurrency when
     // forwarding Alternator request between shards - if necessary for LWT.
     smp_service_group _ssg;
-
+    lw_shared_ptr<expression_cache_t> _expression_cache;
```

Comment:
Ah, I think lw_shared_ptr is because of the abandoned futures. This isn't good, we're piling on problems instead of fixing them.

Reply:
I agree. In a month from now (or even today), nobody will remember the reason for these tricks, and the next person to touch this code will break it. Comments can help a bit, but even better is not to use such tricks.